sale_margin_delivered/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Margin Delivered",
+    "version": "16.0.1.0.0",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/margin-analysis",
+    "category": "Sales",
+    "license": "AGPL-3",
+    "depends": ["sale_stock", "sale_margin_security"],
```

Comment:
```suggestion
    "depends": ["sale_stock"],
```

Please do not force to install module that are not required.

Reply:
But you need to depend on this module to be able to apply the permissions of the "sale_margin_security.group_sale_margin_security" group in the "sale_margin_delivered.view_order_form" view. Isn't it?

Reply:
can be a glue module. something like ``sale_margin_delivered_security``
- module A : sale_margin_security deploy a restriction to hide margin to some users
- module B : sale_margin_delivered displays real delivered margin. (other feature).

user can want feature B without feature A. 
Making both modules working together can be done with a glue module IMO.

Reply:
I agree with what you say Sylvain, someone who has the need to have those modules separately should feel free to make those changes.

Reply:
ok ! So I'll propose a PR against this PR.

Reply:
Done here : https://github.com/Tecnativa/margin-analysis/pull/2